src/Header/Header.less

```diff
@@ -87,6 +83,9 @@
 		.moon-header-title-below,
 		.moon-header-sub-title-below {
 			line-height: 39px;
+			.enyo-locale-non-latin & {
+				line-height: @moon-non-latin-medium-header-line-height;
```

Comment:
Sorry @DavidUM, I should have been more clear. I think it's inappropriate to use the *medium header's* line height to set the line height of the `title-below` and `sub-title-below`. They aren't logically related and don't even share the same font.